misk/src/main/kotlin/misk/tracing/TracingModule.kt

```diff
@@ -4,13 +4,16 @@ import misk.inject.KAbstractModule
 import misk.logging.getLogger
 import misk.tracing.backends.jaeger.JaegerBackendModule
 import misk.tracing.backends.noop.NoopTracerBackendModule
+import misk.tracing.backends.zipkin.ZipkinBackendModule
 
 private val logger = getLogger<TracingModule>()
 
 class TracingModule(val config: TracingConfig) : KAbstractModule() {
   override fun configure() {
     if (config.backends.jaeger != null) {
```

Comment:
supernit: At this point might want to switch over the a ``when`` statement. e.g.

````kotlin
when {
  config.backends.jaeger != null -> install(JaegerBackendModule(config.backends.jaeger))
  config.backends.zipkin != null -> install(ZipkinBackendModule(config.backends.zipkin))
  else -> {
    install(NoopTracerBackendModule())
   logger.info("No tracing backend configured. NoopTracer installed")
  }
}
````